charts/zenko/requirements.yaml

```diff
@@ -20,6 +20,10 @@ dependencies:
 - name: redis-ha
   version: "2.1.2"
   repository: "https://kubernetes-charts.storage.googleapis.com/"
+- name: grafana
+  version: "1.11.6"
+  repository: "https://kubernetes-charts.storage.googleapis.com/"
+  condition: grafana.enabled
```

Comment:
Why make this conditional? Prometheus deployment isn't conditional either, and there's not much use for Prometheus without Grafana.